src/reducers/scrollStateHelper.js

```diff
@@ -104,6 +106,8 @@ function _getRowAtEndPosition(state, rowIndex) {
   // We need to update enough rows above the selected one to be sure that when
   // we scroll to selected position all rows between first shown and selected
   // one have most recent heights computed and will not resize
+  // NOTE (jordan) We don't need to update leading buffered rows here because
+  // _updateHeightsAboveViewport updates scrollY accordingly
```

Comment:
This is interesting.  We need to update the row height for everything in the viewport, but don't do the leading buffered rows.  Works out fine because we update scrollY when we update the leading buffered rows.  We need to update those in viewport though so we can find the correct first row which makes our target row the last row.